src/main/resources/application-local.yaml

```diff
@@ -9,8 +9,8 @@ spring:
   datasource:
     driver-class-name: org.postgresql.Driver
     url: jdbc:postgresql://localhost:5432/refunds
-    username: refunds
-    password: refunds
+    username: ${POSTGRES_USERNAME}
+    password: ${POSTGRES_PASSWORD}
     properties:
```

Comment:
username: ${POSTGRES_USERNAME}
password: ${POSTGRES_PASSWORD}
Check if the above can be used in docker-compose.yml line 32&33

Reply:
No Arbab. We shouldnt use even dummy password